tools/updatePackageJsonVersion.R

```diff
@@ -1,5 +1,5 @@
 # Updates the package.json `version` to match the DESCRIPTION `Version`
-pkg <- jsonlite::read_json("package.json", simplifyVector = TRUE)
```

Comment:
The `simplifyVector = TRUE` here was causing the package.json file to be changed in a way that broke it.